src/routes/docs/quick-starts/ruby/+page.markdoc

```diff
@@ -0,0 +1,212 @@
+---
+layout: article
+title: Start with Ruby
+description: Dive into our step-by-step guide on integrating Appwrite with your Ruby server backend application. Get your backend up and running quickly with this tutorial.
+difficulty: beginner
+readtime: 5
+back: /docs/quick-starts
+---
+Learn how to setup your first Ruby project powered by Appwrite.
+{% section #step-1 step=1 title="Create project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console).
+
+If this is your first time using Appwrite, create an account and create your first project.
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+Then, under **Integrate with your server**, add an **API Key** with the following scopes.
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/integrate-server.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/integrate-server.png)
+{% /only_light %}
+
+| Category  {% width=120 %} | Required scopes       | Purpose |
+|-----------|-----------------------|---------|
+| Database  | `databases.write`     | Allows API key to create, update, and delete [databases](/docs/products/databases/databases). |
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to read [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% /section %}
+{% section #step-2 step=2 title="Create Ruby project" %}
+Create a Ruby CLI application.
+
+```sh
+mkdir my-app
+cd my-app
+bundle init
+```
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+
+Install the Ruby Appwrite SDK.
+
+```sh
+gem install appwrite
+```
+{% /section %}
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier. 
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Create a new file `app.ruby` and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```ruby
+# Initialize the Appwrite client
+require 'appwrite'
+
+client = Appwrite::Client.new()
```

Comment:
If we don't include the Appwrite module, we'll have to prefix every class access with `Appwrite::`

```suggestion
require 'appwrite'

include Appwrite

client = Client.new()
```

Reply:
Done